kubernetes/template/DeploymentTemplates/azuredeploy.json

```diff
@@ -127,6 +127,17 @@
       "metadata": {
         "description": "The base for Kubernetes images"
       }
+    },
+    "nodeDistro": {
+      "defaultValue": "ubuntu",
+      "allowedValues": [
+        "ubuntu",
+        "aks"
+      ],
+      "metadata": {
+        "description": "Node distro to be used to deploy Kubernetes on Azure Stack. The value could be ubuntu or aks."
```

Comment:
Unless an explanation is required, I wouldn't include valid values in the field description, it is harder to maintain.

Reply:
Updated.